### DISCUSSION TITLE
The Legend of Zelda: Breath of the Wild

### DISCUSSION THREAD

**Discussion by iiQx2** [skyline-emu/skyline#506] [Game Requests]
Zelda is the best game! ~~And i have one question! Can you do skyline emulator in ios? No matter how hard it is DO IT! Even if a jailbreak is required and it was very difficult, please do it. I want you to sweep the iOS system and I thank you for everything you provide for Android and I hope to see you in the iOS system💖💖.~~

**Comment by PixelyIon** [comment]
BOTW is fairly high on the list of games we want running ourselves, as for iOS that topic was explored in #387. Do limit further discussion of any potential iOS port to that discussion rather than this game request.

**Comment by WINDROID-EMU** [comment]
claro que e o zelda ne

**Reply by lucasferrr** [reply]
pior que é mesmo

**Comment by OpalaSS** [comment]
Melhor jogo do mundo.

**Reply by lucasferrr** [reply]
concordo

**Comment by ykarorian2003** [comment]
ykarorian2003@gmail.com

**Comment by 7aychu23** [comment]
Zelda is the best trust me 😁
